cookbook/product_asset/sftp.rst

```diff
@@ -0,0 +1,91 @@
+How to connect to an external server for storage
+================================================
+
+Quick Overview
+--------------
+
+**These cookbook examples are made for asset catalog storage but it works the same way for other catalogs**
+
+This cookbook assumes that you already configured an external server. This cookbook should be made before the installation of the pim. Else you will have to migrate your data to your external server.
+
+Connect to an sftp server
+-------------------------
+
+Lets assume that you already have a fully functional sftp server ready to receive files. You will have to setup the adapter in the app/config/config.yml file.
+
+.. code-block:: yaml
+
+    # /app/config/config.yml
+    oneup_flysystem:
+        adapters:
+            asset_storage_adapter:
+                sftp:
+                    host: ftp.domain.com
+                    port: ~
+                    username: ~
+                    password: ~
+                    root: ~
+                    timeout: ~
+                    privateKey: ~
+                    permPrivate: ~
+                    permPublic: ~
+
+Don't forget to add your sftp credentials.
```

Comment:
add a note to say something like:

If you want to change the way your product media are stored, you have to configure the key `XXX` instead of `asset_storage_adapter`.